### ISSUE TITLE
Diff colors not working properly?

### ISSUE THREAD

**Issue by triplejam** [git-cola/git-cola#757]
I don't think setting cola.color.text with `git config --global cola.color.text 31363b` or manually in your .gitconfig actually is working
`[cola "color"]
  text = 31363b
	add = 11d116
	remove = ed1515
`
It's in the manual. But it doesn't work for me. It would be nicer if you could set these as an environmental variable like icons. I am on a debian based distro with qtcurve

**Comment by davvid** [comment]
What does it show in the `Help -> About` dialog's `Version` tab?  The quickest way there is to run `git cola about` and click on the `Version` tab.

what does it show if you do `git config cola.color.text`?  I only ask because the line above in your example is missing newlines, but that might just be a github comment formatting thing.  I just tested it by editing `.git/config` in a repo and adding the colors from above and it seems to be working here, so I'd be curious to know why it's not working in your setup.  I'm on debian/testing.

Let's see if we can dig into what's going on.  `env GIT_COLA_TRACE=full git cola` might be one way to debug it since it should report exactly what we're getting back from `git`, but it is a bit noisy.

The config values are read by `cola/widgets/diff.py` once during startup, so you typically need to restart after making changes.  Printing the `self.color_text`, `color_add`, and `color_remove` variables might provide some insight into what's going on.  Likewise, the `color()` method in `cola/gitcfg.py` might be another place to audit how the colors are interpreted.

**Comment by triplejam** [comment]
Hmm... I tried it again today and now it appears to be working. The relevant lines from the trace are `31363bcola.color.add
11d116cola.color.remove
ed1515' ''
`

I don't know how to force a new line in the markdown.
I also used --global with the git config.
`Git Cola version 3.0 
Linux-4.13.0-1-amd64-x86_64-with-debian-9.2 
Python (/usr/bin/python3) 3.6.3 (default, Oct 3 2017, 21:16:13) [GCC 7.2.0] 
Git 2.15.0 
Qt 4.8.7 
QtPy 1.3.1 
PyQt4 (API v2) 4.12.1 `
